README.md

```diff
@@ -75,8 +75,8 @@ matching dependencies:
 ```
 cd $GOPATH/src/github.com/decred/dcrd
 git pull
-glide install
-go install $(glide nv)
+dep ensure -update
+go install ./...
```

Comment:
This will cause issues on versions of Go that are still supported but do not ignore the vendor directory when provided `./...`.